system/Session/Handlers/BaseHandler.php

```diff
@@ -102,13 +114,18 @@ abstract class BaseHandler implements SessionHandlerInterface
 
     public function __construct(AppConfig $config, string $ipAddress)
     {
+        /** @var SessionConfig $session */
+        $session = config('Session');
```

Comment:
@MGatner Why is it a breaking change?
Without `app/Config/Session.php` or `system/Config/Session.php`, `$session` will be `null`.
And `$this->cookieName` will be `$config->sessionCookieName` or `$this->cookieName`.
Am I something wrong?

Reply:
No, you're correct! I didn't realize bull coalescence would allow property access on a non-object, but apparently it does: https://3v4l.org/pmAoE

I still think this would be cleaner the other way, with presence of the Session config as the determinant instead of checking each property on a potentially-nonexistent object. But it isn't a breaking change, which was the important part.

Reply:
Okay. I agree with @MGatner .
`if ($session = config('Session')) { ...` is cleaner.

Reply:
The same approach was used to create the `Config\Cookie` and `Config\Security` classes